src/aspire/source/__init__.py

```diff
@@ -316,16 +300,32 @@ def cache(self, im=None):
             im = self.images()
         self._im = im
 
-    def images(self, start=0, num=np.inf, indices=None, batch_size=256, *args, **kwargs):
+    def images(self, start=0, num=np.inf, indices=None, batch_size=32678, *args, **kwargs):
```

Comment:
Didn't we ultimately decide to remove `batch_size` and also remove the default values for `start` and `num` (forcing the user to commit to the number of images they want)?